foundation/foundation-mda/src/main/resources/profiles.json

```diff
@@ -989,29 +992,6 @@
 			}
 		]
 	},
-	{
-		"name": "aissemble-hive-metastore-db-kubernetes",
```

Comment:
I: Deleting unused profiles

Reply:
Q: Did this result in an update to the generator comments somewhere for these files? Not seeing it in the PR.

Reply:
Why not? The underlying file has been deleted.

Reply:
That's actually a fair point.  I'll go ahead and delete them.

Reply:
Done